java-vtl-script/src/main/java/no/ssb/vtl/script/visitors/ExpressionVisitor.java

```diff
@@ -130,43 +107,33 @@ public VTLExpression visitUnaryExpr(VTLParser.UnaryExprContext ctx) {
         }
     }
 
+    private static boolean isNull(VTLTyped typed) {
```

Comment:
Is the method name correct?

Reply:
Yes, this actually shows that we should have a "null" type under the object type.

Reply:
Does that mean if an object is an instance of VTLObject, then it is a "null" object? If so, then you're right, this should be more explicit.